bin/aapp_dr_runner.py

```diff
@@ -262,7 +253,7 @@ def move_aapp_log_files(config):
                 base_filename = os.path.basename(file_name)
                 dst = os.path.join(destination, base_filename)
                 shutil.move(file_name, dst)
-            except Exception as err:
+            except Exception:
```

Comment:
Could we be more precise? I think `OSError` would cover it, right? https://docs.python.org/3/library/exceptions.html#os-exceptions

Reply:
Yes, think you are right!